Makefile

```diff
@@ -53,25 +74,22 @@ DEP_DIR := $(OBJ_DIR)/dep
 # Subdirectory for log files
 LOG_DIR := log
 
+# Directory for coverage report
+COV_DIR := .vscode/coverage
+
 # ******************************
 # *     Vars for compiling     *
 # ******************************
 
-CXX := c++
+CXX := clang++
```

Comment:
Do we know that the 42 pcs are using clang as compilers?

Reply:
Tested it right now: yes.

But this was an oversight: I had it in there to check if it makes a difference. Reverted back to c++. Also subject states we have to use c++

However: we should still try to compile on both clang and gcc. Maybe the link gets changed or something?